easybuild/easyblocks/i/ipp.py

```diff
@@ -109,12 +109,12 @@ def make_module_req_guess(self):
         guesses = super(EB_ipp, self).make_module_req_guess()
 
         if LooseVersion(self.version) >= LooseVersion('9.0'):
-            lib_path = os.path.join('lib', self.arch)
+            lib_path = [os.path.join('ipp/lib', self.arch), os.path.join('lib', self.arch)]
             include_path = 'ipp/include'
```

Comment:
While we're at it. Use os.path.join all the way, i.e.
lib_path = [os.path.join('ipp', 'lib', self.arch), os.path.join('lib', self.arch)]
include_path = os.path.join('ipp', 'include')